javascript/house-style.md

```diff
@@ -560,14 +560,14 @@ The JavaScript in your application should consist of one or more entry points an
 - A module which handles all analytics events.
 - A module which implements sticky header functionality.
 
-If you are exporting only one thing, use `export default`. If you are exporting more than one thing, use __only__ named exports. You should not export both default and named exports from the same module. 
+Your module should use named exports for all exports. 
```

Comment:
could this benefit from a "we do this/we _don't_ do this" example?

Reply:
Good plan 🦇 👨